tests/conftest.py

```diff
@@ -86,6 +86,7 @@
 
 def pytest_addoption(parser):
     parser.addoption("--testbed", action="store", default=None, help="testbed name")
+    parser.addoption("--log_dir_with_testbed_idx", action="store", default=None, help="testbed name index")
```

Comment:
Is log_dir used in other place? If no, let's just name this parameter as --log_dir, and pass testbed_idx outsidely

Reply:
not yet, will use --log_dir